packages/document/docs/en/api/javascript-api/instance.mdx

```diff
@@ -267,6 +267,124 @@ await rsbuild.startDevServer({
 });
 ```
 
+## rsbuild.createDevServer
+
+Rsbuild comes with a built-in dev server designed to improve the development experience. When you run the `rsbuild dev` command, the server will start, providing features such as page preview, routing, and hot module reloading.
+
+If you want to integrate Rsbuild DevServer into an existing Server, you can use this method to obtain the instance methods of Rsbuild DevServer and call them on demand.
+
+- **type:**
+
+```ts
+type RsbuildDevServer = {
+  /** start the Rsbuild DevServer */
+  listen: () => Promise<{
+    urls: string[];
+    port: number;
+    server: Server;
+  }>;
+
+  /** The following APIs will be used when you use a custom server */
+
+  /** The resolved rsbuild server config */
+  config: {
+    devServerConfig: DevConfig & ServerConfig;
+    port: number;
+    host: string;
+    https: boolean;
+    defaultRoutes: Routes;
```

Comment:
What is the `default` mean? When will this param be used

Reply:
removed